idea/idea-frontend-fir/idea-fir-low-level-api/src/org/jetbrains/kotlin/idea/fir/low/level/api/lazy/resolve/FirLazyDeclarationResolver.kt

```diff
@@ -56,7 +56,7 @@ internal class FirLazyDeclarationResolver(
         val firFile = declaration.getContainingFile()
             ?: error("FirFile was not found for\n${declaration.render()}")
         val provider = firFile.session.firIdeProvider
-        val fromPhase = if (reresolveFile) declaration.resolvePhase else minOf(firFile.resolvePhase, declaration.resolvePhase)
+        fun fromPhase() = if (reresolveFile) declaration.resolvePhase else minOf(firFile.resolvePhase, declaration.resolvePhase)
```

Comment:
Maybe out of scope this PR/commit, but was it intentional to say `reresolveFile`? Like resolving file again?

Reply:
I think that's intentional. Maybe it's better to call it `resolveFileAgain` instead? I will let JetBrains' folks chime in on this one.

Reply:
Yeap, it's intentional, better to rename to something more clear